common/primitives/Cargo.toml

```diff
@@ -16,8 +16,8 @@ targets = ['x86_64-unknown-linux-gnu']
 codec = { package = "parity-scale-codec", version = "3.1.5", default-features = false, features = [
```

Comment:
Parity is using 3.4.0 for 0.9.42

```
name = "parity-scale-codec"
version = "3.4.0"
```

Reply:
I double-checked against substrate and found the version to be 3.2.2. It has been updated throughout.